src/Xamarin.Android.Build.Tasks/Tasks/GenerateLayoutCodeBehind.cs

```diff
@@ -121,7 +122,7 @@ public override bool Execute ()
 
 		void GetLineInfo (IXmlLineInfo linfo, out int line, out int column)
 		{
-			if (linfo == null || linfo.HasLineInfo ()) {
+			if (linfo == null || !linfo.HasLineInfo ()) {
```

Comment:
I half wonder if this should be a separate PR.

Reply:
Nah. It's a one-liner and really doesn't deserve special treatment.